### PR TITLE
Versionv1 update

### PR DESCRIPTION
**Pull request by ironoa**

### CODE REVIEW

**Comment by ironoa** [comment]
https://github.com/w3f/infrastructure/issues/278

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "polkadot-deployer",
-  "version": "1.3.2",
+  "version": "1.3.3",
```
Comment (ironoa) [inline package.json:3]: or 1.4.0 ?

Comment (ironoa) [reply package.json:3]: could this be seen differently because of the "substrate-telemetry": "v2.2.0" upgrade ?

Comment (fgimenez) [reply package.json:3]: i think we are not breaking backwards compatibility, but we are introducing new features, as i see it we should bump minor, wdyt?

**Pull request merged**